exercises/practice/leap/.approaches/if-expression/content.md

```diff
@@ -0,0 +1,57 @@
+# If expresion
+
+```elm
+isLeapYear : Int -> Bool
+isLeapYear year =
+  let
+    divisbleBy number = 
+      modBy number year == 0 
+  in
+    if divisbleBy(400) then
+      return True
+    
+    else if divisbleBy(100) then
+      return False
+    
+    else if divisbleBy(4) then
```

Comment:
I'm not sure that `if something then True else False` is ever better than `something`, especially in Elm where there is no truthiness.